packages/bot-web-ui/src/components/help-contents/flyout-help-base.jsx

```diff
@@ -72,16 +72,14 @@ const HelpBase = ({
             {!is_search_flyout && (
                 <div className='flyout__help-footer'>
                     <Button
-                        className='flyout__button-previous'
-                        has_effect
+                        className='btn--secondary flyout__button-previous'
```

Comment:
We have a `secondary` prop in `button.jsx`, could we use this vs adding the class ourselves?